etc/bash_completion.d/juju-2.0

```diff
@@ -335,13 +339,26 @@ _juju_complete_2_0() {
 }
 # _juju_2_0_cache_TTL [mins]
 export _juju_2_0_cache_TTL=2
+
 # All above completion is juju-2.0 specific, uses $_juju_cmd_JUJU_2_0
-export _juju_cmd_JUJU_2_0=/usr/bin/juju-2.0
+
+# Detect juju built from source (highest priority)
+if [ -x "$GOPATH/bin/juju" ]; then 
+    export _juju_cmd_JUJU_2_0="$GOPATH/bin/juju"
```

Comment:
This seems good. Might mean developers see packaging issues less, but in exchange will be catching their own problems instead.